### DISCUSSION TITLE
Welcome to matplotlib Discussions!

### DISCUSSION THREAD

**Discussion by milos-korenciak** [milos-korenciak/matplotlib#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GerfriedC** [comment]
Hello, I'm running into problems using matplotlib in a python script started from the command line, which is executing a python script loaded from a private website- after some hours org.gtk.fvs.Metadata is activated and then the system activates ca.descr.dconf and then  the debian on raspberry hangs completely.
Thanks

**Reply by milos-korenciak** [reply]
GerfriedC Hi gefried - this is not official matplotlib release.
You probably want to write into this project: https://github.com/matplotlib/matplotlib
This is just my personal fork which is going to be deleted soon.
